perftracker/templates/comparison_id.html

```diff
@@ -335,7 +395,78 @@ <h4>Job: {{ job }}</h4>
     $(window).on('DOMContentLoaded load resize scroll', delayed_render);
     $('.collapse').on("shown.bs.collapse hidden.bs.collapse", delayed_render);
     delayed_render();
+
+    $('.panel__btn_toggle').click(function () {
```

Comment:
this function may be reduced and simplified like:
`panel.toggleClass('panel_pinned');`
`panel.is(":hidden")? showPanel() : hidePanel();`